app/javascript/packs/sinon.js

```diff
@@ -0,0 +1,3 @@
+  // Sinon is used to fake the browser's time when used with Timecop
+  import sinon from "sinon/pkg/sinon"
```

Comment:
I don't think we really want this extra dependency at this point. Is it possible to remove this?

Reply:
Also, since we have users from all over the country using the app on a shared instance, my gut says we should avoid baking the assumption of matching timezones into the application.

Reply:
The sinon dependency is how we tell the Capybara browser that Timecop has altered the current time.  We are using Sinon to provide Timecop functionality into Capybara. This dependency only exists for the "test" environment, and is not required for either development or production environments. The clue to this approach was that the Timecop date was set to `June 1, 2018`, but the screenshots showed that the browser had picked a date range of `Jan 1, 2019 - Dec 31, 2019`. This is because the Rails server and the Capybara browser disagree on when `now` is.

The alternative to this would be to make all the Timecop time travels do their magic in a way that does not break Capybara.  This is tricky because it has to be done carefully in every place that Timecop is used. It is also likely to trip up other developers in the future who may not be aware of this requirement.

Reply:
> Also, since we have users from all over the country using the app on a shared instance, my gut says we should avoid baking the assumption of matching timezones into the application.

They don't have to match, but the tests become non-deterministic. The browser tests rely on the browser's local time to determine how to fill in the date range picker. At certain times of day, these will result in different days being calculated (server thinks "today is monday" but browser thinks today is sunday"), which will break the tests. Fixing this would require refactoring the application to give users a timezone attribute and altering the queries to take timezone into account, which is more than I wanted to take on in this PR. I acknowledge it's a bug, but I think these tests have value even if we aren't addressing the fine details of timezone yet.

Reply:
Yeah, honestly it wouldn't be hard to rip out if we change our minds so I'm good to try it out and see what we think